copier/tools.py

```diff
@@ -90,7 +92,23 @@ def printf_exception(
         print(HLINE, file=sys.stderr)
 
 
-def cast_str_to_bool(value: Any) -> bool:
+def cast_to_str(value: Any) -> str:
+    """Parse anything to str.
+
+    Params:
+        value:
+            Anything to be casted to a str.
+    """
+    if isinstance(value, str):
+        return value.value if isinstance(value, Enum) else value
+    if isinstance(value, (float, int, Decimal)):
+        return str(value)
+    if isinstance(value, (bytes, bytearray)):
+        return value.decode()
+    raise ValueError(f"Could not convert {value} to string")
```

Comment:
How about this?

```suggestion
    return repr(value)
```

Reply:
This would let non-string-like things like `object()` pass validation as `"<object object at 0xfa2598>"` which I think should fail instead because `object()` is not a proper answer to a `type: str` question. IMO, if somebody really wants to pass the `__repr__` value of an object, then they should pass it explicitly by calling `repr(...) explicitly/consciously. WDYT, @yajo?

Reply:
Of course. Thanks for the explanation!